apps/api/internal/mcp/connections_test.go

```
@@ -0,0 +1,672 @@
+			"is a CLAIM that the organisation has no connections, and this read "+
+			"never happened", len(got))
+	}
+}
+
+func TestListConnectionsHandlerAnswersNon2xxOnFailureWithNoConnectionsKey(t *testing.T) {
+	tenantID := uuid.New()
+	store := &fakeStore{grantsErr: errors.New("database is down")}
+	eng := newConnectionsRouter(t, store, orgPrincipal(tenantID))
+
+	w := httptest.NewRecorder()
+	eng.ServeHTTP(w, httptest.NewRequest(http.MethodGet, ConnectionsPath, nil))
```

golangci-lint (2.12.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)